src/pages/home/sidebar/OptionRow.js

```diff
@@ -138,9 +139,23 @@ const OptionRow = ({
                     </TouchableOpacity>
                 </View>
             )}
-            {option.hasDraftComment && (
-                <Icon src={Pencil} width={variables.fontSizeSmall} height={variables.iconSizeSmall} />
-            )}
+            <View style={styles.flexRow}>
+                {option.hasDraftComment && (
+                    <View style={styles.ml2}>
+                        <Icon src={Pencil} width={variables.iconSizeNormal} height={variables.iconSizeNormal} />
+                    </View>
+                )}
+                {option.isPinned && (
+                    <View style={styles.ml2}>
+                        <Icon
+                            src={PinCircle}
+                            width={variables.iconSizeNormal}
```

Comment:
I think you might be able to omit these declarations as `iconSizeNormal` is the default size of each icon. So could you try without `width`, `height`, and `fill` and see if it looks the same?

Reply:
Yes, removed these properties and it looks the same 👍